other/blacklistipchecker.py

```diff
@@ -0,0 +1,101 @@
+######################################
+# This CLI script checks if entered ips are or aren`t blacklisted on different lists
+# After configuration it can even send an email report
+# You need to get API key on blacklistchecker to make it work (it`s free 30 IPs per month)
+######################################
+
+
+import requests
+import sys
+from IPy import IP
+import smtplib
+from email.message import EmailMessage
+import json
+
+API_key = ''
+subject = ''
+from = ''
+to = ''
+smtp_user = ''
+smtp_passw = ''
+smtp_serv = ''
+smtp_port = 587
+
+def test_ip(ip):
+    detects = []
+    result = requests.get('https://api.blacklistchecker.com/check/'+ip, auth=(API_Key,''))
+    result_dec = json.loads(result.content)
+    print(result_dec)
+    if result_dec.get('statusCode'):
+        res="limit exceeded"
+        detects = "!"
+        return(res, detects)
+    elif result_dec.get('detections') != 0:
+        res = ip
+        blsts = result_dec.get('blacklists')
+        for n in blsts:
+            if n.get('detected') == True:
+                detects.append(n.get('name'))
+        return(res, detects)
+    else:
+        return()
+
+res_txt = 'IP blacklists check returned next results:\n'
+send_param = 0
+if len(sys.argv) > 1:
+    for x in sys.argv[1:]:
+        if x in '-y-Y':
+            send_param = 1
+        else:
+            try:
+                IP(x)
+            except:
+                print('Argument isn`t a valid ip('+x+")\n Skipping argument")
+            else:
+                if test_ip(x):
+                    res, detec = test_ip(x)
+                    if res == "limit exceeded":
+                        res_txt = res_txt +" LIMIT Exceeded!"
+                        break
+                    else:
+                        res_txt = res_txt + str(res) +' is detected in '+ str(detec) +' blacklists \n \n'
+else:
+    print('Type in ip to check')
+    x = input()
+    try:
+        IP(x)
+    except:
```

Comment:
AGAIN??

Reply:
IPy literally works that way. I can easily fix it with 
except Exception as ex:
    print('Failed \n" + ex + 'Finishing \n")

Reply:
https://realpython.com/the-most-diabolical-python-antipattern

Reply:
Got it. I can fix it by using logging module. What should i do next?

Reply:
Click `details` next to each of the failing ❌ tests below and try to get them to be green ✅